docs/proposals/PROPOSAL_A.md

```diff
@@ -0,0 +1,68 @@
+# Proposal A - Runtime Annotations
+
+This proposal recommends changes that allow runtimes to quickly select the appropriate image from an index using the platform and annotations on the descriptor.
+
+## Links
+
+| Description | Link |
+|-------------|------|
+| Image Index Spec | <https://github.com/opencontainers/image-spec/blob/main/image-index.md> |
+
+## Modifications
+
+### Image Spec
+
+The following language is already included in the Image Index definition:
+
+> If multiple manifests match a client or runtime's requirements, the first matching entry SHOULD be used.
+
+This would add a section for "Annotations":
+
+> Runtimes MAY be configured to prefer manifests with specific annotations.
+
+An example would also be added to the Image Index:
+
+```jsonc
+{
+  "schemaVersion": 2,
+  "mediaType": "application/vnd.oci.image.index.v1+json",
+  "manifests": [
+    {
+      "mediaType": "application/vnd.oci.image.manifest.v1+json",
+      "size": 7143,
+      "digest": "sha256:e692418e4cbaf90ca69d05a66403747baa33ee08806650b51fab815ad7fc331f",
+      "platform": {
+        "architecture": "amd64",
+        "os": "linux"
+      }
+    },
+    {
+      "mediaType": "application/vnd.oci.image.manifest.v1+json",
+      "size": 7682,
+      "digest": "sha256:5b0bcabd1ed22e9fb1310cf6c2dec7cdef19f0ad69efa1f392e94a4333501270",
+      "platform": {
+        "architecture": "amd64",
+        "os": "linux"
+      },
+      "annotation": {
+        "com.example.gpu": "example-gpu-model"
+      }
+    }
+  ]
+}
+```
+
+In the above scenario, the generic `linux/amd64` entry is listed first and should be used by default.
+Runtimes MAY be configured to prefer the `com.example.gpu: example-gpu-model` entry to run the image customized for their specific environment.
+
+Image authors SHOULD limit annotations to those needed to differentiate descriptors or for runtimes requirements.
```

Comment:
There shouldn't be any reason that annotations could be used for other things, although then it's again very wild westy. I guess we can't avoid that, it's a pro and a con!

Reply:
SHOULD may be the wrong word here. It's designed to be guidance because all too often people see a button and insist on pressing it without knowing how it will break things downstream. (This goes back to the artifact manifest debate in OCI.)